ocfweb/stats/semester_job.py

```diff
@@ -0,0 +1,89 @@
+import ocflib.printing.quota as quota
+from django.http import HttpResponse
+from matplotlib.figure import Figure
+from matplotlib.ticker import MaxNLocator
+from ocflib.lab.stats import semester_dates
+from ocflib.printing.quota import WEEKDAY_QUOTA
+from ocflib.printing.quota import WEEKEND_QUOTA
+
+from ocfweb.component.graph import canonical_graph
+from ocfweb.component.graph import plot_to_image_bytes
+
+
+@canonical_graph(default_start_end=semester_dates)
+def weekday_jobs_image(request, start_day, end_day):
+    return HttpResponse(
+        plot_to_image_bytes(get_jobs_plot('weekday', start_day, end_day), format='svg'),
+        content_type='image/svg+xml',
+    )
+
+
+@canonical_graph(default_start_end=semester_dates)
+def weekend_jobs_image(request, start_day, end_day):
+    return HttpResponse(
+        plot_to_image_bytes(get_jobs_plot('weekend', start_day, end_day), format='svg'),
+        content_type='image/svg+xml',
+    )
+
+
+# dictionary that gives parameters/config of different job graphs
+graphs = {
+    'weekday':
+        {'query': '''
+                SELECT `pages`, SUM(`count`) AS `count`
+                FROM `public_jobs`
+                WHERE
+                    (`pages` <= %s) AND
+                    (DAYOFWEEK(`day`) MOD 7 > 1) AND
+                    (`day` BETWEEN CAST(%s AS Date) AND CAST(%s AS DATE))
+                GROUP BY `pages`''',
+         'quota': tuple([WEEKDAY_QUOTA]),
+         'title': 'Semester Weekday Jobs Distributiom'},
+    'weekend':
+        {'query': '''
+                SELECT `pages`, SUM(`count`) AS `count`
+                FROM `public_jobs`
+                WHERE
+                    (`pages` <= %s) AND
+                    (DAYOFWEEK(`day`) MOD 7 <= 1) AND
+                    (`day` BETWEEN CAST(%s AS Date) AND CAST(%s AS DATE))
+                GROUP BY `pages`''',
+         'quota': tuple([WEEKEND_QUOTA]),
+         'title': 'Semester Weekend Jobs Distribution'}
+}
+
+
+def freq_plot(data, title,
+              ylab='Number of Jobs Printed'):
+    """takes in data, title, and ylab and makes a histogram, with
+    the 1:len(data) as the xaxis
+    """
+    fig = Figure(figsize=(10, 4))
+    ax = fig.add_subplot(1, 1, 1)
+
+    tickLocations = range(1, len(data) + 1)
+    width = 0.8
+    ax.bar(tickLocations, data, width, color='blue')
```

Comment:
I would leave `color` at default so it matches with the existing graphs on the page.